recipe/meta.yaml

```diff
@@ -33,12 +33,12 @@ requirements:
     - libtool   # [not win]
     - m2-patch  # [win]
   host:
-    - ncurses   # [not win]
-    - readline  # [not win]
-    - zlib      # [not win]
+    - ncurses   6.4  # [not win]
+    - readline  {{readline}} # [not win]
+    - zlib      {{zlib}}     # [not win]
   run:
-    - readline  # [not win]
-    - zlib      # [not win]
+    - {{ pin_compatible('readline') }}  # [not win]
+    - {{ pin_compatible('zlib') }}      # [not win]
```

Comment:
@katietz I wonder why we didn't use pin_compatible earlier? Are there any drawbacks?

Reply:
well, pin compatible has the disadvantage of relying on the pretty random run_export of the used readline package.  So having this stuff more explicit in recipe is benefitial, as more easy to understand and reliable